.github/workflows/build.yml

```diff
@@ -8,21 +8,20 @@ on:
 
 jobs:
   build-docker-images:
-    strategy:
-      matrix:
-        dolibarr_version: [ 15.0.3, 16.0.5, 17.0.4, 18.0.6, 19.0.4, 20.0.4, 21.0.0, develop ]
```

Comment:
Can you please add back the strategy matrix for dolibarr_version?

We are down from ~40mins to ~5mins build time for the whole version because it parallelise the build process.

See: https://github.com/Dolibarr/dolibarr-docker/actions/runs/14159712921